Sources/NIO/EventLoop.swift

```diff
@@ -50,6 +50,83 @@ public struct Scheduled<T> {
     }
 }
 
+/// Returned once a task was scheduled to be repeatedly executed on the `EventLoop`.
+///
+/// A `RepeatedTask` allows the user to `cancel()` the repeated scheduling of further tasks.
+public final class RepeatedTask {
+    private let delay: TimeAmount
+    private let eventLoop: EventLoop
+    private var scheduled: Scheduled<EventLoopFuture<Void>>?
+    private var task: (() throws -> EventLoopFuture<Void>)?
```

Comment:
I think we removed the need for this to be `throws`, didn't we? It doesn't actually manifest in the word `try` anywhere in the code for this object, but let's constrain the type further and not allow it to throw.